src/static/client/app/login/login.scss

```diff
@@ -1,5 +1,11 @@
 @import '../common.scss';
 
+.login, .signup {
+  display: flex;
+  justify-content: center; // Horizontally center children
+  align-items: center;     // Vertically center children
+  height: 100vh;           // Cover the entire viewport height
+
   .button-container {
     display: flex;
     flex-direction: column;
```

Comment:
What are you using this for? Buttons should center themselves by default, so flex usually isn't necessary for inside of buttons, unless they are picture buttons or something like that.